Foundation/NSDictionary.swift

```diff
@@ -122,7 +122,8 @@ public class NSDictionary : NSObject, NSCopying, NSMutableCopying, NSSecureCodin
     }
     
     public override convenience init() {
-        self.init(objects: nil, forKeys: nil, count: 0)
+        // FIXME: This isn't really safe. Either the required initializer handles optional pointers, or it doesn't.
+        self.init(objects: unsafeBitCast(nil as UnsafePointer<AnyObject>?, to: UnsafePointer<AnyObject>.self), forKeys: unsafeBitCast(nil as UnsafePointer<NSObject>?, to: UnsafePointer<NSObject>.self), count: 0)
```

Comment:
can we use placeholder memory here too? e.g. a static UnsafePointer<Void> bit pattern or something like NSData and promise that it will never be de-referenced internally? We are breaking subclassing patterns by doing this btw; internally the values are nullable and externally vended they are not.